### DISCUSSION TITLE
ansible-lint not finding all ansible related yml files

### DISCUSSION THREAD

**Discussion by evulhotdog** [ansible/ansible-lint#1727] [Q&A]
I utilize ansible-pull, ansible-lint does not appear to be catching all of the files that it should be. 

I have different components laid out is mostly by application or function. This is the root of the folder, with `local.yml`. It does, however, pick up the two yml files in the <root>/handlers folder.  

- Is it possible that the linter is not picking up components properly because of the folder naming? 
- If so, what is the right folder structure?

![image](https://user-images.githubusercontent.com/365456/133848220-52a29694-9b2d-4fab-bfda-d69b30ae970a.png)

**Comment by webknjaz** [comment]
What's the command you use to run it? Which files do you expect it to find?

**Reply by evulhotdog** [reply]
I've used both:
`ansible-lint -v --force-color -p local.yml`
and
`ansible-lint -v --force-color -p` for auto discovery, which actually finds _less_ when it's scanning all the files.

**Reply by webknjaz** [reply]
The question stands: which files do you want to be found? And how are they used?

**Reply by evulhotdog** [reply]
Apologies, was a little late when I answered your question, and missed some!

These are what I'd like it to pick up, however, as I mentioned, it _only_ picks up the one I specify, and the ones in the /handlers/ dir.

```
./local.yml
./templates/vulnerability-patching/vulnerability-patching.yml
./templates/session-manager/session-manager.yml
./templates/certificates/certificates.yml
./templates/ansible/ansible-pull.yml
./templates/ansible/ansible_pull-configs.yml
./templates/datadog/datadog-plugins.yml
./templates/datadog/datadog.yaml
./templates/sumologic/sumologic.yml
./templates/image-downloader/image-downloader-ansible.yml
./templates/high-availability/high-availability.yml
./handlers/restart_mirth.yml
./handlers/set_up_mysql.yml
```
